concourse/steps/draft_release.mako

```diff
@@ -46,9 +51,26 @@ processed_version = version.process_version(
 )
 
 repo_dir = ci.util.existing_dir('${repo.resource_name()}')
-component_descriptor_v2 = cm.ComponentDescriptor.from_dict(
-    ci.util.parse_yaml_file('${component_descriptor_v2_path}'),
-)
+
+have_ctf = os.path.exists('${ctf_path}')
+have_cd = os.path.exists('${component_descriptor_v2_path}')
+if not have_ctf ^ have_cd:
+    ci.util.fail('exactly one of component-descriptor, or ctf-archive must exist')
+elif have_cd:
+    component_descriptor_v2 = cm.ComponentDescriptor.from_dict(
+        ci.util.parse_yaml_file('${component_descriptor_v2_path}'),
+    )
+elif have_ctf:
+    component_descriptors = list(cnudie.util.component_descriptors_from_ctf_archive(
+        '${ctf_path}',
+    ))
+    if not component_descriptors:
+        ci.util.fail(f'No component descriptor found in CTF archive at ${ctf_path})
+    if len(component_descriptors) > 1:
```

Comment:
can we not handle more than one?

Reply:
currently we also cannot handle more the one in the release trait.
I would keep it like this and open another PR with the support for multiple component descriptors in the release and draft release PR?